mobile/src/screens/chat-layout/chat-layout.tsx

```diff
@@ -0,0 +1,136 @@
+import React from 'react';
+
+import { ScrollView, Text, View } from '#libs/components/components';
+import {
+  useAppForm,
+  useCallback,
+  useEffect,
+  useRef,
+  useState,
+} from '#libs/hooks/hooks';
+
+import { ChatInput, MessageItem } from './components/components';
+import { PREVIOUS_USER } from './libs/constants';
+import { styles } from './styles';
+
+type Message = {
+  id: number;
+  isUser: boolean;
+  message: string;
+};
+
+const ChatLayout: React.FC = () => {
```

Comment:
let's rename it to Chat (don't forget about folder/file name)
```suggestion
const Chat: React.FC = () => {
```

Reply:
I kept the folder name unchanged because the 'chat' folder already exists in task #54

Reply:
ok, let's leave for now, but after it's merged I think we will need separate Stack navigation for Chat module and we will need to rename this components